e2e/rbd.go

```diff
@@ -232,6 +234,15 @@ var _ = Describe("RBD", func() {
 			e2elog.Failf("failed to create node secret with error %v", err)
 		}
 		deployVault(f.ClientSet, deployTimeout)
+
+		kernelRelease, err := util.GetKernelVersion()
```

Comment:
You'll get the kernel version where the e2e (client) executable runs, not the kernel version that is used by the csi-rbdplugin pod.

There are different ways you can get the kernel version of the node(s) where the containers are running:

a. exec into a pod and run `uname`
b. use the k8s API to list the nodes and inspect [the details](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#nodesysteminfo-v1-core)

Reply:
Hmm, good catch Niels, then what we are doing here: https://github.com/ceph/ceph-csi/blob/devel/e2e/rbd.go#L442
might be also wrong then! I will fix this at both places.

Thanks, @nixpanic.

Reply:
Done now!